agot-bg-game-server/src/common/ingame-game-state/westeros-game-state/bidding-game-state/BiddingGameState.ts

```diff
@@ -105,6 +105,8 @@ export default class BiddingGameState<ParentGameState extends BiddingGameStatePa
     }
 
     firstStart(participatingHouses: House[] = []): void {
+        this.bids = new BetterMap<House, number>();
```

Comment:
Why specifically moving this into `firstStart`?

Reply:
I tried to resolve the ux issue that the old bidding is still preselected. But with no success.